api/src/main/java/com/ford/labs/retroquest/metrics/Metrics.java

```diff
@@ -39,40 +36,43 @@ public long getTeamCount(LocalDate startDate, LocalDate endDate) {
 
     @ManagedAttribute
     public int getFeedbackCount() {
-        return feedbackRepository.findAll().size();
+        return getFeedbackCount(null, null);
     }
 
     public int getFeedbackCount(LocalDate startTime, LocalDate endTime) {
-        LocalDateTime finalEndTime = endTime == null ? LocalDateTime.now() : endTime.atStartOfDay();
-        LocalDateTime finalStartTime = startTime == null ? LocalDateTime.MIN : startTime.atStartOfDay();
-        List<Feedback> feedbackList = feedbackRepository.findAll().stream()
-                .filter(feedback -> !feedback.getDateCreated().isBefore(finalStartTime))
-                .filter(feedback -> !feedback.getDateCreated().isAfter(finalEndTime))
-                .collect(toList());
-        return feedbackList.size();
+        LocalDateTime finalEndTime = Optional.ofNullable(endTime)
```

Comment:
if the parameters LocalDate startTime, LocalDate endTime has been wrapped into an Object something named 
DateRange.

All the logic related to calculating finalEndTime ,finalStartTime  can be moved into the class, while this method can happily make Use of it

Reply:
Thanks for the suggestion! We have made this refactor.